client/src/KudosRow.jsx

```diff
@@ -1,16 +1,44 @@
+// KudosRow.jsx
+import { useEffect, useState } from "react";
 import "./KudosRow.css";
 import KudosCard from "./KudosCard";
-const KudosRow = () => {
+
+const KudosRow = ({ searchTerm }) => {
+  const [boardInfo, setBoardInfo] = useState([]);
+
+  const populateCards = async () => {
+    try {
+      const resp = await fetch("http://localhost:3000/kudos/board");
+      const data = await resp.json();
+      setBoardInfo(data);
+    } catch (error) {
+      console.error(error);
+    }
+  };
+
+  useEffect(() => {
+    populateCards();
+  }, [searchTerm]);
```

Comment:
Why do we need this dependency of searchTerm here?

Reply:
I thought to update everytime searchTerm changes but I dont think we need